Lib/getpass.py

```diff
@@ -119,9 +119,9 @@ def win_getpass(prompt='Password: ', stream=None, *, echo_char=None):
             raise KeyboardInterrupt
         if c == '\b':
             if echo_char and pw:
-                msvcrt.putch('\b')
-                msvcrt.putch(' ')
-                msvcrt.putch('\b')
+                msvcrt.putwch('\b')
+                msvcrt.putwch(' ')
```

Comment:
We can use putch here right?

Reply:
NVM, let's use putwch as well, it's more readable at the cost of a slightly slower call I guess.